### DISCUSSION TITLE
Share a volume (RWX) between deployments

### DISCUSSION THREAD

**Discussion by aessing** [longhorn/longhorn#6567] [Q&A]
Hi,

I'm now trying for weeks, but couldn't find a solution. I would like to share a Longhorn Volume between two deployments.

1. Application in a Statefulset
2. code-seerver to allow changes of files on the volume of the application running in the StatefulSet

From my understanding, that would be possible with RWX, but I couldn't find how... I couldn't get this working and tutorials are mostly without longhorn (using direct NFS).

Can you help me and explain how to accomplish that, if possible?

Thanks so much
Andre

**Comment by c3y1huang** [comment]
The [rwx-nginx-deployment.yaml](https://github.com/longhorn/longhorn/blob/v1.5.1/examples/rwx/rwx-nginx-deployment.yaml#L19-L20) in Longhorn's examples might help.
